.travis.yml

```diff
@@ -3,10 +3,17 @@ language: php
 php:
     - 5.5
     - 5.6
+    - 7.0
     - hhvm
 
+matrix:
+    include:
+        - php: 5.5
+          env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+
 before_script:
-    - travis_retry composer install --no-interaction
+    - travis_retry composer self-update
```

Comment:
I agree with @rdohms, you was a bit more "aggressive" than you should have be. Apart from that, the idea was to update, then to suggest a manual update. Actually the warning is rather a "do-what-you-want" solution.